app/controllers/scc_accounts_controller.rb

```diff
@@ -7,7 +7,7 @@ class SccAccountsController < ApplicationController
   # GET /scc_accounts
   def index
     @scc_accounts = resource_base.search_for(params[:search], order: params[:order])
-                                 .paginate(page: params[:page])
+                                 .paginate(page: params[:page]).where(organization: Organization.current)
```

Comment:
I think you should add the filter for the organization, before the pagination (maybe even before the actual search).

Otherwise, the pages might have varying numbers of elements, depending on how much of the accounts of that page belong to the current organuization

Reply:
Thank you for your feedback. I moved the filter. I tested the pagination and it was actually broken before, so I fixed it as well.